daemon/src/auto_rollover.rs

```diff
@@ -108,7 +107,9 @@ where
             .expect("actor to be able to give address to itself");
         for id in cfd_ids {
             try_continue!(async {
-                let cfd = load_cfd(id, &mut conn).await?;
+                #[allow(deprecated)] // This checking for rollover eligibility does not fit into the command abstraction (yet). Figure out a way for fixing it.
```

Comment:
a missing `TODO`?

Reply:
The fact that we are using `allow(deprecated)` should be enough of a hint that this needs fixing, no? :D

Reply:
I like making such comments clear with `TODO` or `FIXME` (the latter being disfavoured here). Easier to grep or display within the IDE (not sure whether CLion supports that, but magit in emacs shows all the TODOs - so they're harder to ignore). 

I can see how `allow(deprecated)` can be looked the same way - perhaps it's because I'm still quite fresh in this ecosystem....Perhaps I should just add a filter for `allow(deprecated)` in my magit-todos 🤔

No need to address it if you wish to keep it as-is :)